### PR TITLE
Reduce Docker image size

### LINKED ISSUE #64: Clean Docker image
Some issues I noticed with the current Docker image

- [x] something is installing Python 2.7
- [x] why are we overwriting uvicorn / fastapi from the base image by installing from `requirements.txt`
- [x] Clean up unneeded packages after build

### PR DESCRIPTION
**Pull request by disrupted**
Closes #64

### CODE REVIEW

**Comment by disrupted** [comment]
down to 465 MB from 576 MB (~20%)

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
 COPY ./frontend /frontend
-RUN mkdir -p /app/static && npm install --prefix /frontend && npm run build --prefix /frontend && mv /frontend/build/* /app/static/ && rm -rf /frontend
\ No newline at end of file
+RUN mkdir -p /app/static && \
+    npm install --prefix /frontend && \
+    npm run build --prefix /frontend && \
+    mv /frontend/build/* /app/static/ && \
+    rm -rf /frontend
+
+RUN apt-get -y purge --auto-remove -o APT::AutoRemove::RecommendsImportant=false && \
```
Comment (philipp94831) [inline Dockerfile:23]: Can this be run earlier? after the first apt stuff?

Comment (disrupted) [reply Dockerfile:23]: Does it have an advantage to run it earlier? Otherwise I think it's safer & perhaps more effective to clean after the build.

Comment (disrupted) [reply Dockerfile:23]: for Python 2.7 it definitely doesn't work to remove it earlier. It seems npm requires it for the build for some reason

Comment (philipp94831) [reply Dockerfile:23]: Then it is fine. Removing earlier would have been beneficial for docker build layers and caching

**Review by philipp94831 — APPROVED** [review]

**Review by VictorKuenstler — APPROVED** [review]

**Pull request merged**